src/tools/pub/automated-publishing.md

```diff
@@ -0,0 +1,719 @@
+* The `--include-email` option requests that `pub.dev` recognize the
+  service account.
+
+Once you create the `id_token`, Google Cloud writes it to a file that
+resides in a _volume_. Google Cloud uses this mechanism to
+[pass data between steps][16].
+Don't store the token in `/workspace`.
+Google Cloud checks out your repository into `/workspace`.
+If you store the token in `/workspace`, your published package includes
+the `id_token`.
+
+
+### Creating a Cloud Build trigger
+
+With a service account configured and a `cloudbuild.yaml` file in
+the repository, use [console.cloud.google.com](https://console.cloud.google.com/cloud-build/triggers)
+dashboard to create a _Cloud Build Trigger_.
+To create a build trigger, you need to connect to a _source repository_
+and specify which events should trigger a build. You can use [GitHub][18],
+[Cloud Source Repository][19], or one of the [other options][20].
+To learn how to configure a _Cloud Build Trigger_, check out
+[creating and managing build triggers][21].
+
+To publish packages using the Cloud Build Configuration file you created:
+
+1. Open the **Triggers** page in the Google Cloud console.
+1. Select your project from the project selector drop-down menu at the
+   top of the page.
+1. Click **Open**.
+1. If you have an existing trigger, open it. 
+   If you need a new trigger, [create it][28].
+1. Scroll to the **Configuration** section.
+1. Click **Cloud Build Configuration**.
+
+    ![Configuration for trigger](gcb-trigger-configuration.png)
+
+1. Do **not** specify a **Service account** to trigger the build.
+   Let Cloud use its default service account.
+
+
+{{site.alert.note}}
+  You can configure the Cloud Build trigger to run under a custom
+  _service account_. If you want to do this, create a new service
+  account for this purpose. Allow this service account to impersonate
+  the `pub-dev@$PROJECT_ID.iam.gserviceaccount.com` account which can
+  publish to pub.dev.
+
+  In the configuration of the `pub-dev@$PROJECT_ID.iam.gserviceaccount.com`
+  service account, the default Cloud Build service account
+  `$PROJECT_NUMBER@cloudbuild.gserviceaccount.com` can impersonate the
+  `pub-dev@$PROJECT_ID.iam.gserviceaccount.com` service account.
+
+  To learn more about custom service accounts for running Cloud Builds,
+  check out [Configuring user-specified service accounts][22].
+{{site.alert.end}}
+
+When configuring your Cloud Build Trigger, consider who can trigger the
+build. _Anyone who can trigger the build can publish your package._
+Consider allowing only manual builds or use
+[Cloud Build approvals][17] to gate builds as outlined in next section.
+
+### Hardening security with Cloud Build Approvals
+
+When configuring a Cloud Build Trigger, you can select
+**require approval before build executes**. If a Cloud Build Trigger
+requires approval, it won't run when triggered. It waits for approval.
+Use this to limit who can publish new versions of your package.
```

Comment:
> Use this to limit who can publish new versions of your package.

Strong statement,

You may, you can, but "use" -- that's strong -- there are lots of reasonable setups where you don't do this.
It's an option available to you for hardening security. Not a requirement for security. Maybe your repository is lock down sufficiently.
Maybe you're only using manually triggered publishing, etc.